frontend/isoko/src/pages/ListBusiness.tsx

```diff
@@ -215,10 +270,43 @@ const ListBusiness: React.FC = () => {
       Sat: '',
    });
 
+   // This function parses hours input and makes it form ready
+   const parseHours = () => {
+      const x = {
+         Sun: '',
+         Mon: '',
+         Tue: '',
+         Wed: '',
+         Thu: '',
+         Fri: '',
+         Sat: '',
+      };
+      const y = {
+         Sun: '',
+         Mon: '',
+         Tue: '',
+         Wed: '',
+         Thu: '',
+         Fri: '',
+         Sat: '',
+      };
+      console.log('PH: ', props.hours);
+      if (props.hours !== undefined) {
+         Object.entries(props.hours).forEach((entry) => {
+            const [key, value] = entry;
+            x[key] = value.split('-')[0];
+            y[key] = value.split('-')[1];
+         });
+         setOpenHours(x);
+         setCloseHours(y);
+      }
```

Comment:
not sure if you already have this on the ListBusiness page, but might be worth doing a quick validation that they've entered an opening AND closing value

Reply:
props.hours (if included) will automatically determine both of these values. This function just sets the initial state of the component, so I dont think validation is needed here.